### PR TITLE
Localize main pages English content to Hindi

### PR DESCRIPTION
**Pull request by Shailin**
Issue no. 8

1. Updated the content shown in 'Reading and Writing' and 'Math' to Hindi text.

### CODE REVIEW

File: EnglishWebRoot/js/viewSubject.js [hunk 1]
```diff
@@ -224,9 +238,16 @@ document.addEventListener("deviceready", function () {
         ret.cols = [];
         ret.cols.push({gravity: 0.05});
         var attrs = "";
+        var subjectName = ['पढ़ाई और लिखाई','गणित'];
```
Comment (ankitk20) [inline EnglishWebRoot/js/viewSubject.js]: Use " for consistency

**Review by ankitk20 — CHANGES_REQUESTED** [review]

File: EnglishWebRoot/js/viewSubject.js [hunk 2]
```diff
@@ -121,6 +121,10 @@ document.addEventListener("deviceready", function () {
         }
     }
     function unitHTML(level) {
+        var levelNameEng = ['Level 1','Level 2','Level 3','Level 4'];
```
Comment (akashdabhi03) [inline EnglishWebRoot/js/viewSubject.js]: levelNamesEnglish

**Review by akashdabhi03 — CHANGES_REQUESTED** [review]

File: EnglishWebRoot/js/viewSubject.js (hunk 1 above)
Comment (Shailin) [reply EnglishWebRoot/js/viewSubject.js]: Updated

File: EnglishWebRoot/js/viewSubject.js (hunk 2 above)
Comment (Shailin) [reply EnglishWebRoot/js/viewSubject.js]: Updated

**Review by ankitk20 — DISMISSED** [review]

**Review by akashdabhi03 — DISMISSED** [review]

File: EnglishWebRoot/js/viewSubject.js [hunk 3]
```diff
@@ -160,10 +164,20 @@ document.addEventListener("deviceready", function () {
             classes += " unlocked";
         }
 
+        for(var iCount = 0; iCount < levelNameEnglish.length; iCount++){
```
Comment (mathewnik90) [inline EnglishWebRoot/js/viewSubject.js]: Add some comment, mentioning that we have assumed English to be base way to structure the content and that we are overriding language to be specific

Comment (Shailin) [reply EnglishWebRoot/js/viewSubject.js]: Added comments

**Review by ankitk20 — DISMISSED** [review]

**Review by mathewnik90 — APPROVED** [review]

**Pull request merged**